bok_choy/browser.py

```diff
@@ -127,10 +127,15 @@ def browser(tags=None, proxy=None):
     There are three cases:
 
     1. Local browsers: If the proper environment variables are not all set for the second case,
-        then we use a local browser.  The environment variable `SELENIUM_BROWSER` can be set to
-        specify which local browser to use, but the default is firefox.  Additionally, if a proxy
+        then we use a local browser.
+
+        * The environment variable `SELENIUM_BROWSER` can be set to
+        specify which local browser to use, but the default is firefox.
```

Comment:
As long as you are moving around this sentence, I'd change the "but" to an "and" or make that 2 sentences like "...which local browser to use. The default is Firefox."